packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -173,6 +187,68 @@ const makeStartMeetingNotificationLookup: Record<
   poker: makeGenericStartMeetingNotification
 }
 
+const addStandupResponsesToThread = async (
+  res: PostMessageResponse,
+  standupResponses: Array<{user: User; response: TeamPromptResponse}> | null,
+  team: Team,
+  meeting: Meeting,
+  notificationChannel: NotificationChannel
+) => {
+  if (!standupResponses || standupResponses.length === 0) {
+    const message = 'No responses were submitted'
+    const threadRes = await notifySlack(
+      notificationChannel,
+      'meetingEnd',
+      team.id,
+      message,
+      message,
+      res.ts
+    )
+    if ('error' in threadRes) {
+      return handleError(threadRes, team.id, notificationChannel)
+    }
+    return 'success'
+  }
+
+  const results = await Promise.all(
+    standupResponses.map(async ({user, response}) => {
+      const options = {
+        searchParams: {
+          utm_source: 'slack standup summary',
+          utm_medium: 'product',
+          utm_campaign: 'standups',
```

Comment:
@tianrunhe PTAL at these UTMs!

Reply:
Aviva [suggested](https://parabol.slack.com/archives/C03C3J42WDU/p1691109288833119?thread_ts=1691095523.822949&cid=C03C3J42WDU) to change the campaign to `after-meeting`

Reply:
@tianrunhe updated!